src/main/java/de/medizininformatik_initiative/process/report/service/DownloadSearchBundle.java

```diff
@@ -53,7 +53,7 @@ protected void doExecute(DelegateExecution execution, Variables variables)
 		Task task = variables.getStartTask();
 		Target target = variables.getTarget();
 		String searchBundleIdentifier = ConstantsReport.CODESYSTEM_REPORT + "|"
-				+ ConstantsReport.CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE;
+				+ ConstantsReport.CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE_V2;
```

Comment:
If possible reuse existing constant and only change value.
```suggestion
				+ ConstantsReport.CODESYSTEM_REPORT_VALUE_SEARCH_BUNDLE;
```

Reply:
Implemented via dd46ec2.